modules/adkernelBidAdapter.js

```diff
@@ -199,10 +199,16 @@ function getLanguage() {
  */
 function createSite() {
   var location = utils.getTopWindowLocation();
-  return {
+  let result = {
```

Comment:
Please use http://prebid.org/dev-docs/bidder-adaptor.html#referrers instead of `utils.getTopWindowLocation();
 `. We will deprecation this function and `utils.getTopWindowReferrer()` soon

Reply:
@jaiminpanchal27 Done. Unfortunately there is a circleci build failure in unrelevant (I beleive) unit test.